apps/nestjs-wallet/src/vc-api/credentials/types/verification-result.ts

```diff
@@ -0,0 +1,22 @@
+import { IsString, IsArray } from 'class-validator';
```

Comment:
Unused?

Reply:
Nice catch, thanks. Need to add some tooling to catch this 🔖